src/analysis/AppLauncher.ts

```diff
@@ -127,77 +148,85 @@ const ApplicationLauncher = _.flow(
           showButtons: false,
         },
         [
-          Utils.cond(
-            // if user has more than one outdated rstudio analysis, display plural phrasing
-            [
-              _.size(outdatedAnalyses) > 1,
-              () => [
-                p([
-                  'These R files are being edited by another user and your versions are now outdated. Your files will no longer sync with the workspace bucket.',
-                ]),
-                p([getAnalysesDisplayList(outdatedAnalyses)]),
-                p(['You can']),
-                p(['1) ', strong(['save your changes as new copies']), ' of your files which will enable file syncing on the copies']),
-                p([strong(['or'])]),
-                p([
-                  '2) ',
-                  strong(['continue working on your versions']),
-                  ` of ${getAnalysesDisplayList(outdatedAnalyses)} with file syncing disabled.`,
-                ]),
+          _.merge(
```

Comment:
all this big diff is doing is merging two arrays into one because Modal's ts contract takes one arg, not multiple